crypto/ed25519/ed25519.go

```diff
@@ -22,7 +29,10 @@ type Keypair struct {
 	private *PrivateKey
 }
 
+// PrivateKey is the ed25519 Private Key
 type PrivateKey ed25519.PrivateKey
+
+// PublicKey is the ed25519 Public Key
```

Comment:
I feel like comments like these aren't helpful, are they required for the linter to pass? or is it mostly for the godocs? if it's just the latter, I would prefer not having redundant/self explanatory comments. however, if everyone else prefers them, that's ok too

Reply:
The comments are required for the lint to pass, please check the .golangci.yaml lines: 269, 273. I have added it there because there was just too much exported methods and variables without comments and that is required by the linter. All exported fields and methods should be commented its the common practice for Go.

I also added the "error strings should not be capitalized or end with punctuation or a newline" to skip the error messages, we should update this also at a later PR.

the: "and that stutters; consider calling this" and "which can be annoying to use" are also a little tricky to update, there is some exported methods that are returning non exported struct, so we have to either export the struct or not have the method exported.

Reply:
okay, that makes sense. sure, let's open an issue for the error messages.